profiles/coreos/ppc64le/generic/package.use

```diff
@@ -0,0 +1,8 @@
+# Enable SELinux for amd64 targets
+coreos-base/coreos	    selinux
```

Comment:
Does selinux work on ppc64le? Or even if it does perhaps save this for a later PR, cross-compiling selinux is problematic iirc.